### DISCUSSION TITLE
is animated GIF assets resize supported?

### DISCUSSION THREAD

**Discussion by anhzf** [directus/directus#14919] [Archive]
I have GIF assets then I tried to request a thumbnail file like png files, but it's not worked. It's not yet supported or I'm wrong?

for example:
❌ example.com/assets/\<id-of-gif-file>?width=500
It's not worked 🤔

**Comment by br41nslug** [comment]
The transformations will only output the following formats:
`format — What file format to return the thumbnail in. One of jpg, png, webp, tiff`
I'm assuming that these are filtered as input too as transforming an animated gif into one of these static formats would be problematic.

**Comment by rijkvanzanten** [comment]
Transforming GIFs (and keeping them animated) is not supported at this time

**Comment by anhzf** [comment]
Thank you for the answer!

**Comment by SaschaBoerngen** [comment]
Any update in this matter? Can GIF be integrated now?
